### PR TITLE
Add line overload view

### PR DESCRIPTION
**Pull request by JacquesBorsenberger**

### CODE REVIEW

**Review by geofjamg — CHANGES_REQUESTED** [review]
You should virtualize your table. On a large network, and using a very low threshold you have many constraints and switch from nominal voltage mode to overload mode take several seconds because of table rendering.
Also there is a strange glitch, when you scroll the table with the mouse, the table seems to blink.

**Comment by geofjamg** [comment]
image
Maybe when there is few constraints, we should avoid displaying the scrollbar?
The first column with equipment ID is not readable enough, maybe we can have a text background (just behind the text) a little bit light (or dark depending on the theme).
@mikatheo what do you think about this?

**Comment by JacquesBorsenberger** [comment]
table is virtualized and scrollbar is only present if needed.

File: src/redux/local-storage.js [hunk 1]
```diff
@@ -24,6 +24,11 @@ export const getLocalStorageBoolean = (key, defValue) => {
     return value === 'true';
 };
 
+export const getLocalStorageInteger = (key, defValue) => {
```
Comment (geofjamg) [inline src/redux/local-storage.js:27]: See jon comment on usefulness of this function 
https://github.com/gridsuite/gridstudy-app/pull/193#discussion_r490432841

File: src/components/network/overloadedLinesView.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+                        width: 70,
+                    },
+                    {
+                        label: intl.formatMessage({ id: 'Intensity' }),
+                        dataKey: 'intensity',
+                        numeric: true,
+                        width: 70,
+                    },
+                    {
+                        label: intl.formatMessage({ id: 'Limit' }),
```
Comment (geofjamg) [inline src/components/network/overloadedLinesView.js:147]: There is no translation for Limit with an upper case

**Review by geofjamg — CHANGES_REQUESTED** [review]

File: src/components/network/overloadedLinesView.js (hunk 2 above)
Comment (JacquesBorsenberger) [reply src/components/network/overloadedLinesView.js:147]: translation added

**Review by geofjamg — APPROVED** [review]

**Pull request merged**